kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/templates/kroxylicious/KroxyliciousConfigMapTemplates.java

```diff
@@ -75,7 +77,10 @@ private static String buildEncryptionFilter(TestKmsFacade<?, ?, ?> testKmsFacade
                     selector: TemplateKekSelector
                     selectorConfig:
                       template: "KEK_${topicName}"
-                """.formatted(testKmsFacade.getKmsServiceClass().getSimpleName(), getNestedYaml(testKmsFacade.getKmsServiceConfig(), 6));
+                    experimental:
+                      %s
+                """.formatted(testKmsFacade.getKmsServiceClass().getSimpleName(), getNestedYaml(testKmsFacade.getKmsServiceConfig(), 6),
```

Comment:
getNestedYaml! Scream!  But I'll keep my eyes closed.  we need a better way to construct our configs.